tests/test_taxi_service_forms.py

```diff
@@ -0,0 +1,131 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Car, Manufacturer
+
+
+class CarTest(TestCase):
+    def setUp(self) -> None:
+        self.user = get_user_model().objects.create_user(
+            username="admin.user",
+            license_number="ADM12345",
+            first_name="Admin",
+            last_name="User",
+            password="1qazcde3"
+        )
+        self.client.force_login(self.user)
+        self.manufacturer = Manufacturer.objects.create(
+            name="Lincoln",
+            country="USA",
+        )
+
+    def test_create_car(self):
+        response = self.client.post(
+            reverse("taxi:car-create"),
+            {
+                "model": "Continental",
+                "manufacturer": self.manufacturer.id,
+                "drivers": [self.user.id]
+            }
+        )
+        self.assertEqual(response.status_code, 302)
+        self.assertEqual(
+            Car.objects.get(id=self.user.cars.first().id).model,
+            "Continental"
+        )
+
+    def test_update_car(self):
+        self.car = Car.objects.create(
```

Comment:
```suggestion
        car = Car.objects.create(
```

Reply:
Self is not needed here, this variable should exist only inside this local scope